app/helpers/days_helper.rb

```diff
@@ -19,4 +19,10 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def completed_students(activity)
+    completed = cohort.students.select { |student| student.completed_activity?(activity) }.count
+    total = cohort.students.count 
```

Comment:
You can just get the length of the array rather than doing a count query on the database. This produces many redundant database queries.